.github/tools/cibw_repair_wheel_command_windows.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -e
```

Comment:
Out of curiosity, what's happening here?

Reply:
I borrowed that file from another Github project that had a very similar package setup.  I cannot take credit for anything there.  For those two lines you highlighted:

https://askubuntu.com/questions/1402718/is-the-same-usr-bin-env-bash-than-bin-bash

https://linuxcommand.org/lc3_man_pages/seth.html

Reply:
Ah, okay.  I meant the whole file, but now I see that those lines are quite highlighted from this view.

Reply:
Oh, sorry.  There is some problems with bundling a compiled library in a python wheel in windows and this file is supposed to bundle it outside of `pip install`